package.json

```diff
@@ -60,6 +60,7 @@
     "minimist": "1.2.5",
     "n-readlines": "1.0.0",
     "outdent": "0.7.1",
+    "parse-srcset": "ikatyang/parse-srcset#54eb9c1cb21db5c62b4d0e275d7249516df6f0ee",
```

Comment:
What about countries where github banned? They exist :smile:

Reply:
Already banned https://github.com/prettier/prettier/blob/master/package.json#L32

Reply:
To be honestly we should fix it in the future